core/src/test/java/org/onehippo/forge/gallerymagick/core/command/AbstractImageMagickCommandTest.java

```diff
@@ -41,6 +41,9 @@
             if (!_convertCommandAvailable && new File("/usr/local/bin/convert").isFile()) {
                 _convertCommandAvailable = true;
             }
+            if (!_convertCommandAvailable && new File("/opt/local/bin/convert").isFile()) {
+                _convertCommandAvailable = true;
+            }
```

Comment:
@lachire L#44-46 looks the same as #41-43. Could you remove them?

Reply:
same as above